src/avcodec/bitstream.rs

```diff
@@ -0,0 +1,143 @@
+use std::{ffi::CStr, ptr};
+
+// See https://blogs.gentoo.org/lu_zero/2016/03/21/bitstream-filtering/
+
+use crate::{
+    error::{Result, RsmpegError},
+    ffi,
+    shared::*,
+};
+
+wrap_ref!(AVBitStreamFilter: ffi::AVBitStreamFilter);
+
+impl AVBitStreamFilter {
+    /// Find a bitstream filter instance with it's short name.
+    ///
+    /// See [`ffi::av_bsf_get_by_name`] for more info.
+    pub fn find_by_name(name: &CStr) -> Option<AVBitStreamFilterRef> {
+        unsafe { ffi::av_bsf_get_by_name(name.as_ptr()) }
+            .upgrade()
+            .map(|x| unsafe { AVBitStreamFilterRef::from_raw(x) })
+    }
+
+    // TODO: implement ffi::av_bsf_next
+    // const AVBitStreamFilter *av_bsf_next(void **opaque);
```

Comment:
You can create an iterator here. But it takes sometime. If you find it difficult, I'll do it in the following PR.

Reply:
Honestly I don't need this functionality so I'll leave it to you :) lol